src/main/webapp/script.js

```diff
@@ -30,10 +31,18 @@ function getOmdbItem() {
         const omdbItemEntry = $('#omdbItemEntry');
         omdbItemEntry.empty();
 
+        const itemSubmissionButton = $('#itemSubmissionButton');
```

Comment:
nit: `itemSubmissionButton` can be renamed to `submitButton`. Its more concise and still descriptive.

Reply:
OK I renamed button in last commit.